src/wallet/feebumper.cpp

```diff
@@ -66,7 +66,7 @@ bool CFeeBumper::preconditionChecks(const CWallet *pWallet, const CWalletTx& wtx
     return true;
 }
 
-CFeeBumper::CFeeBumper(const CWallet *pWallet, const uint256 txidIn, int newConfirmTarget, bool ignoreGlobalPayTxFee, CAmount totalFee, bool newTxReplaceable)
+CFeeBumper::CFeeBumper(const CWallet *pWallet, const uint256& txidIn, int newConfirmTarget, bool ignoreGlobalPayTxFee, CAmount totalFee, bool newTxReplaceable)
```

Comment:
```txidIn``` should not be const nor pass-by-reference.
If ```CFeeBumper``` is going to cannibalize ```txidIn``` two lines down it should be copied in the case it was an lvalue. In the case that it was an rvalue, it will not be copied anyway.
```const uint256&``` only causes inefficiencies (due to unnecessary copying) when the caller sends ```txidIn``` as an rvalue